prepare-vms/settings/kube101.html

```diff
@@ -98,7 +97,7 @@
             </table>
         </p>
         <p>You can find the slides at:
-            <center>{{ url }}</center>
+            <center>{{ slides_url }}</center>
```

Comment:
I know that some people want to use a different URL on the cards, because the cards are typically small and cannot fit long URLs.